src/main/java/com/listywave/user/application/service/UserService.java

```diff
@@ -178,4 +183,20 @@ public void updateListVisibility(Long loginUserId, Long listId, Boolean beforeIs
 
         list.updateVisibility(!beforeIsPublic);
     }
+
+    @Transactional(readOnly = true)
+    public UserElasticSearchResponse searchUserByElastic(Long loginUserId, String search, Pageable pageable) {
```

Comment:
`Long loginUserId` 앞에 `@Nullable` 추가해주세요!

```suggestion
    public UserElasticSearchResponse searchUserByElastic(@Nullable Long loginUserId, String search, Pageable pageable) {
```